draft-ietf-quic-recovery.md

```diff
@@ -244,11 +250,11 @@ ack-eliciting packet. QUIC recovery algorithms do not assume the peer sends
 an ACK immediately when receiving a second ack-eliciting packet.
 
 In order to accelerate loss recovery and reduce timeouts, the receiver SHOULD
-send an immediate ACK when it receives a new packet which is not one greater
-than the largest received packet number. A receiver MAY send immediate ACKs
-for the next few ack-eliciting packets that are received, but SHOULD NOT
-send an immediate ACK for more than 1/8 RTT after receiving an out-of-order
-packet.
+send immediate ACKs for an interval after it receives an out-of-order packet.
```

Comment:
The intent of the text is you SHOULD send one immediate ACK, but you may send more than one immediate ACK.  This new text changes that to you SHOULD send an immediate ACK for an interval, and I don't want to make that a SHOULD, only a MAY.

Reply:
The text allows the interval to be zero, but I agree that's weaselly. I'll clarify more.